src/main/java/com/google/edith/DealItem.java

```diff
@@ -0,0 +1,122 @@
+package com.google.edith;
+
+/** 
```

Comment:
The first asterisk on the line that starts the javadoc comment should align with the other asterisks like:
```
/**
 * Text
 */
```